src/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './GoodsList.scss';
+
+type Props = {
+  goods: string[]
+};
+
+export const GoodsList: React.FC<Props> = ({ goods }) => {
+  return (
+    <>
+      <ul className="list">
+        <h1>List of goods:</h1>
```

Comment:
use h2, there can only be one h1 on the page

```suggestion
        <h2>List of goods:</h2>
```